packages/ember-metal/lib/is_proxy.js

```diff
@@ -1,7 +1,10 @@
-import { symbol } from 'ember-utils';
-
-export const IS_PROXY = symbol('IS_PROXY');
+import { meta as metaFor } from './meta';
 
 export function isProxy(value) {
-  return typeof value === 'object' && value && value[IS_PROXY];
+  if (typeof value === 'object' && value) {
+    let meta = metaFor(value);
```

Comment:
I think we should use `peekMeta` (since there is no need to create a meta if one isn't already found).

Something like:

```js
let meta = peekMeta(value);
return typeof meta === 'object' && meta !== null && meta.isProxy()
```